includes/reports/overview/list-table/class-sensei-reports-overview-list-table-abstract.php

```diff
@@ -342,10 +344,15 @@ class="sensei-date-picker"
 		 * @param {string} $report_type The report type.
 		 */
 		do_action( 'sensei_reports_overview_after_top_filters', $this->type );
-		?>
 
-		<?php
-		submit_button( __( 'Filter', 'sensei-lms' ), '', '', false );
+		$filters_content = ob_get_clean();
+
+		if ( ! empty( trim( $filters_content ) ) ) {
```

Comment:
Is the call to the actions very important here?

If not, I think we could just return early on this method instead when ` $this->data_provider->get_is_last_activity_filter_enabled()` is `true`...

Reply:
> Is the call to the actions very important here?

Yep! It introduces a group filter through Sensei Pro. And maybe some user could also be using it for other customizations?

Reply:
Fair enough! Thanks for the reply!